frontend/src/components/ChatExample.jsx

```
@@ -0,0 +1,409 @@
+    };
+
+    console.log('💡 콘솔에서 window.checkAuth() 실행하여 인증 상태 확인 가능');
+
+    return () => {
+      // cleanup
+      delete window.checkAuth;
+    };
+
+    // URL 파라미터 확인 (소셜 로그인 후 리다이렉트 처리)
+    const urlParams = new URLSearchParams(window.location.search);
+    const needsAdditionalInfo = urlParams.get('additional-info');
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)